### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Locally run_Screen Reader: Jaws is not reading the label names "Main Class Name" & "WinUtils.exe Location"

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3631]
**GitHub Tags**: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ScreenReader; #JAWS; #MAS1.3.1; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 1.3.1

**Repro Steps:**
1)Turn on screen reader (Jaws) 
2)Launch IntelliJ IDEA
3)Install the scala plugin and Tab till Create New Project button and press enter
4)Tab till Azure Spark/HDInsight-->Spark Project with Sample (Scala)
5)Tab till Build Tools-->Maven
6)Tab till Next button and press enter
7)Fill the required details & Tab till finish button and press enter
8)Give the code in the console by navigating to myApp (project Name)> src > main > scala >New-->Scala Class. Navigate to Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp
9)Fill the Name and Main class name in Run/Debug Configurations dialog 
10)Tab till Locally run tab and press enter
11)Tab till  label names "Main Class Name" & "WinUtils.exe Location" and verify jaws is reading the label names or not

**Actual Result:**
Jaws is not reading the label names "Main Class Name" & "WinUtils.exe Location"

**Expected Result:**
Jaws should read the label names "Main Class Name" & "WinUtils.exe Location"

**User Impact:**
Visually impaired users cannot fill the information in the text field if screen reader doesn't read the label names

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug_3631_jaws not reading label name.pptx](https://github.com/microsoft/azure-tools-for-java/files/3655969/Bug_3631_jaws.not.reading.label.name.pptx)

**Comment by t-rufang** [comment]
Fixed in #3673

**Comment by InduPriya1805** [comment]
Regressed the bug in the below environment
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IC192.6603.28
The bug is fixed so closing the bug. Please find the attached video for reference.
[Bug_3631_fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3844979/Bug_3631_fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749727
